hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieParquetInputFormat.java

```diff
@@ -105,7 +106,8 @@ private static boolean checkIfHudiTable(final InputSplit split, final JobConf jo
     try {
       Path inputPath = ((FileSplit) split).getPath();
       FileSystem fs = inputPath.getFileSystem(job);
-      HoodieStorage storage = new HoodieHadoopStorage(fs);
+      HoodieStorage storage = HoodieStorageUtils.getStorage(
```

Comment:
We should instead use `HoodieStorageUtils.getStorage(convertToStoragePath(inputPath), HadoopFSUtils.getStorageConf(fs.getConf())`